ginkgo.Jenkinsfile

```diff
@@ -64,6 +65,7 @@ pipeline {
             }
             steps {
                 sh "cd ${TESTDIR}; make tests-ginkgo"
+                sh "go get golang.org/x/tools/cmd/cover && go get github.com/mattn/goveralls && find -iname "*goveralls*" && $HOME/go/bin/goveralls -coverprofile=${TESTDIR}coverage-all.out -service=travis-ci -repotoken=${COVERALLS_TOKEN}"
```

Comment:
I know that is wip, but:
 
- Could we use a docker container?
- Due we run the test using docker-compose, could we create a target in `test/docker-compose.yaml`

Regards